.github/workflows/cibuild.yml

```
@@ -22,7 +22,7 @@ jobs:
 
       matrix:
         platform:
-        - { display_name: 'Linux Native Libraries', os: ubuntu-latest, preset_os: linux }
+        - { display_name: 'Linux Native Libraries', os: ubuntu-24.04, preset_os: linux }
```

```suggestion
        - { display_name: 'Linux Native Libraries', os: ubuntu-22.04, preset_os: linux }
```
Just so some reasonably older distros can work with the artifacts (Debian 12)